frontend/src/bundles/auth/auth-api.ts

```diff
@@ -29,7 +29,7 @@ class AuthApi extends BaseHttpApi {
                 contentType: ContentType.JSON,
                 payload: JSON.stringify(payload),
                 hasAuth: false,
-                query: { referralCode },
+                query: { referralCode: referralCode ?? '' },
```

Comment:
now I'm thinking about why we're even using query parameters for this. Maybe its better to just put this code in body?

Reply:
I extended api to use query in order not to put everything in payload and for further reuse of them, for example, in filters, etc. Is better to remove it at all?

Reply:
I just don't think that passing referrelCode via query parameters is appropriate in this case. It might be more reasonable to send it in the request body.

Reply:
Ok